<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1169</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(44, 46, 48);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="spacing">
       <number>12</number>
      </property>
      <item row="9" column="0">
       <widget class="QPushButton" name="b_launch">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Initiate Takeoff in 2min</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" rowspan="6">
       <widget class="QLCDNumber" name="lcdNumber"/>
      </item>
      <item row="11" column="0">
       <widget class="QPushButton" name="b_ejection">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Chute ejection</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" rowspan="13">
       <widget class="QGraphicsView" name="graphicsView_2"/>
      </item>
      <item row="7" column="2" rowspan="6" colspan="2">
       <widget class="QGraphicsView" name="graphicsView_3"/>
      </item>
      <item row="0" column="0" rowspan="2">
       <widget class="QLabel" name="state">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Rocket Status / Launchpad Status&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QPushButton" name="exit">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QPushButton" name="b_init">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>INIT</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QPushButton" name="b_abort">
        <property name="styleSheet">
         <string notr="true">
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>ABORT</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4" rowspan="7" colspan="2">
       <widget class="QTextEdit" name="console">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
        </property>
       </widget>
      </item>
      <item row="10" column="4" rowspan="3" colspan="2">
       <widget class="PlotWidget" name="graph2" native="true"/>
      </item>
      <item row="7" column="4" rowspan="3" colspan="2">
       <widget class="PlotWidget" name="graph1" native="true"/>
      </item>
      <item row="4" column="2" rowspan="3" colspan="2">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>Kozuka Gothic Pr6N H</family>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 0, 0);</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ignition&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2" rowspan="2" colspan="2">
       <widget class="QLabel" name="hold">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>87</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Kozuka Gothic Pro H</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 170, 0);</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;HOLD&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Rocket State:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Launchpad State:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1169</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>exit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>621</y>
    </hint>
    <hint type="destinationlabel">
     <x>551</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
